common/src/main/java/gov/cms/ab2d/common/model/Contract.java

```diff
@@ -140,7 +141,9 @@ public boolean isAutoUpdatable() {
         return updateMode == UpdateMode.AUTOMATIC;
     }
 
-    public boolean hasDateIssue() {
-        return ContractType.CLASSIC_TEST == contractType;
+
+    public ContractDTO toDTO() {
+        return new ContractDTO(getContractNumber(), getContractName(),
```

Comment:
Calling the getters is not strictly necessary (but does no harm).  The field values can be used directly.